src/widgets/communication/ui/communication.tsx

```diff
@@ -62,14 +62,14 @@ export const Communication = ({ courseName, lang = 'en' }: RequiredProps) => {
           </figure>
           <div>
             <Subtitle text={localizedContent[lang].subTitle} />
-            <Paragraph>
+            <Paragraph size="small" className="paragraph">
```

Comment:
I suggest to change it like in comment above

Reply:
[Resolved](https://github.com/rolling-scopes/site/pull/475/commits/c0158c42b2443716e3e9157717628ee7e2503cb4).